appengine/pom.xml

```diff
@@ -91,6 +91,12 @@
             <artifactId>mysql-connector-java</artifactId>
             <version>8.0.19</version>
         </dependency>
+        <dependency>
```

Comment:
Dropwizard already comes with ObjectMapper, let's use that?

Reply:
Updated code to use ObjectMapper